io/cmd.go

```diff
@@ -0,0 +1,185 @@
+package io
+
+import (
+	"bufio"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"regexp"
+	"strings"
+	"sync"
+)
+
+// Returns the output of the external process.
+// If the output is not needed, use RunCmd
+func RunCmdOutput(config CmdConfig) (string, error) {
+	for k, v := range config.GetEnv() {
+		os.Setenv(k, v)
+	}
+	cmd := config.GetCmd()
+	if config.GetErrWriter() == nil {
+		cmd.Stderr = os.Stderr
+	} else {
+		cmd.Stderr = config.GetErrWriter()
+		defer config.GetErrWriter().Close()
+	}
+	output, err := cmd.Output()
+	return string(output), err
+}
+
+// Runs the external process and prints the output of the process.
```

Comment:
I suggest rephrasing to:
// Runs an external process and prints its output to stdout  / stderr.

Reply:
Done